test/functional/mempool_accept.py

```diff
@@ -70,6 +70,8 @@ def run_test(self):
         node = self.nodes[0]
         self.wallet = MiniWallet(node)
 
+        assert_equal(node.getmempoolinfo()['permitbaremultisig'], False)
```

Comment:
Could have a test with the `True` case?

Reply:
I second this.  It would add value to have tests that cover:
1) permitbaremultisig disabled
2) permitbaremultisig enabled

This might be accomplished through the use of a second node (`self.num_nodes`) or perhaps through stopping the node, modifying configuration with `replace_in_config()`, and restarting it.  Typically num_nodes is to be minimized and start/stops avoided when possible, but I do not immediately see how different `permitbaremultisig` states could be tested without doing one of the two (see https://github.com/bitcoin/bitcoin/blob/3aaf7328eb656b642e5f0f74f3e4d51645a1d0ab/test/functional/README.md?plain=1#L65-L71).  Maybe it's a bit overkill, but we would want to know that the code behind the RPC on bitcoind still works if there are changes in the future.

Reply:
Just adding the true check in `mempool_datacarrier` seems like it would be easy?

Reply:
@ajtowns Good idea, can add test simple way without adding much more code. 84e03ab48e0c832ab2c0d8e6babb6563a0ee92dc